components/org.wso2.carbon.identity.api.server.secret.management/org.wso2.carbon.identity.api.server.secret.management.common/src/main/java/org/wso2/carbon/identity/api/server/secret/management/common/SecretManagementConstants.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org).
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.api.server.secret.management.common;
+
+/**
+ * Secret management related constant class.
+ */
+public class SecretManagementConstants {
+
+    public static final String ERROR_PREFIX = "ECM-";
+
+    public static final String SECRET_CONTEXT_PATH = "/secrets";
+    public static final String CONFIG_MGT_ERROR_CODE_DELIMITER = "_";
```

Comment:
How's `CONFIG_MGT_ERROR_CODE_DELIMITER` related to secret management ?

Reply:
Already resolved with https://github.com/wso2/identity-api-server/pull/303/files